lib/features/hooks/hooks.rb

```diff
@@ -108,28 +123,28 @@
   # TODO Revamp and log sessions
   if scenario.failed?
     STDOUT.puts '^^^ +++'
-    if Server.sessions.empty?
+    if Maze::Server.sessions.empty?
       $logger.info 'No valid sessions received'
     else
-      $logger.info "#{Server.sessions.size} sessions were received:"
-      Server.sessions.all.each.with_index(1) do |request, number|
+      $logger.info "#{Maze::Server.sessions.size} sessions were received:"
+      Maze::Server.sessions.all.each.with_index(1) do |request, number|
         $logger.info "Session #{number}:"
         LogUtil.log_hash(Logger::Severity::INFO, request)
       end
     end
 
-    if Server.errors.empty?
+    if Maze::Server.errors.empty?
       $logger.info 'No valid errors received'
     else
-      $logger.info "#{Server.errors.size} errors were received:"
-      Server.errors.all.each.with_index(1) do |request, number|
+      $logger.info "#{Maze::Server.errors.size} errors were received:"
+      Maze::Server.errors.all.each.with_index(1) do |request, number|
         $logger.info "Request #{number}:"
         LogUtil.log_hash(Logger::Severity::INFO, request)
       end
     end
   end
 
-  next if MazeRunner.config.farm == :none
+  next if MazeRunner.config.farm == :none || !MazeRunner.config.bs_browser.nil?
```

Comment:
Would removing this line & changing the `else` on 154 to `elsif MazeRunner.config.farm != :none` be clearer?

Reply:
Unfortunately that won't work, because the farm for Browsers is still `:bs` and the selenium driver doesn't have the `reset_with_timeout` method.  But it can certainly be clearer and I _think_ a similar change can be made if we test `MazeRunner.config.bs_device`on the `elsif` instead.